qa/rpc-tests/sc_cert_listsinceblock.py

```diff
@@ -0,0 +1,381 @@
+        self.sync_all()
+        print "chain height = {}".format(self.nodes[0].getblockcount())
+
+        mark_logs("Calling listsinceblock on Node2 for h={}, hash={}".format(block_heights_d[2], blocks_d[2]), self.nodes, DEBUG_MODE)
+        ret = self.nodes[2].listsinceblock(blocks_d[2], 1, False, True)
+        #pprint.pprint(ret)
+
+        cert = certs_d[0]
+        # first cert is not there
+        found = False
+        for x in ret['transactions']:
+            if x['txid'] == cert:
+                found = True
+        assert_false(found)
+
+        cert = certs_d[1]
+        # second cert is there, it is mature and it refers to the block where it matured
+        found = False
+        mat_block_hash = self.nodes[2].getblockhash(mat_height_d[1])
+        for x in ret['transactions']:
+            if x['txid'] == cert:
+                found = True
+                assert_equal(x['amount'],   Decimal('2.02'))
+                assert_equal(x['category'], 'receive')
+                assert_equal(x['blockhash'], mat_block_hash)
+                assert_equal(x['isBackwardTransfer'], True)
+        assert_true(found)
+
+        cert = certs_d[2]
+        # last cert is not there
+        found = False
+        for x in ret['transactions']:
+            if x['txid'] == cert:
+                found = True
+        assert_false(found)
+
+        # get the block before the one containing the last certificate
+        h = block_heights_d[2] - 1
+        bl = self.nodes[0].getblockhash(h)
+
+        mark_logs("Calling listsinceblock on Node2 for h={}, hash={}".format(h, bl), self.nodes, DEBUG_MODE)
+        ret = self.nodes[2].listsinceblock(bl, 1, False, True)
+        #pprint.pprint(ret)
+
+        # last cert is there and is immature
+        # we also have one ordinary output from this cert 
+        cert = certs_d[2]
+        found_bwt = False
+        found_out = False
+        for x in ret['transactions']:
+            if x['txid'] == cert:
+                assert_equal(x['blockhash'], blocks_d[2])
+                if 'isBackwardTransfer' in x:
+                    assert_equal(x['category'], 'immature')
+                    assert_equal(x['amount'],   Decimal('3.02'))
+                    assert_equal(x['isBackwardTransfer'], True)
+                    found_bwt = True
+                else:
+                    assert_equal(x['category'], 'receive')
+                    assert_equal(x['amount'],   Decimal('0.003'))
+                    found_out = True
+        assert_true(found_bwt)
+        assert_true(found_out)
+
+        # reach the height of the third certificate, that will make the SC cease, and re-issue the command
+        c = self.nodes[0].getblockcount()
+        mark_logs("Node 0 generates {} block".format(mat_height_d[2] - c),self.nodes,DEBUG_MODE)
+        self.nodes[0].generate(mat_height_d[2] - c)
+        self.sync_all()
+        print "chain height = {}".format(self.nodes[0].getblockcount())
```

Comment:
Use the brackets for `print` for future compatibility with Python 3.

```suggestion
        print("chain height = {}".format(self.nodes[0].getblockcount()))
```